tchannel/tornado/tchannel.py

```diff
@@ -128,6 +131,14 @@ def __init__(self, name, hostport=None, process_name=None,
         # server created from calling listen()
         self._server = None
 
+        # warn if customers are still using this old and soon to be delted api
+        if _from_new_api is False:
+            deprecate(
+                "tchannel.tornado.TChannel is deprecated and will be removed" +
+                " in a a future version - please switch usage to " +
```

Comment:
I'm not confident this will be removed in 0.19.0 - so I'm just saying future version.